swiss-political-science-review.csl

```diff
@@ -87,17 +92,21 @@
   <macro name="access">
     <choose>
       <if type="article-journal" match="any">
-        <group delimiter=" ">
-          <text variable="DOI" prefix="https://doi.org/"/>
-          <group delimiter=" " prefix="[" suffix="]">
-            <text term="accessed"/>
-            <date variable="issued">
-              <date-part name="day" suffix="."/>
-              <date-part name="month" suffix="." form="numeric"/>
-              <date-part name="year"/>
-            </date>
-          </group>
-        </group>
+        <choose>
+          <if match="none" variable="page volume">
+            <group delimiter=" ">
+              <text variable="DOI" prefix="https://doi.org/"/>
+              <group delimiter=" " prefix="[" suffix="]">
+                <text term="accessed"/>
+                <date variable="issued">
```

Comment:
I just want to make sure this is on purpose -- it seems odd to me to use the issued date labelled zugegriffen/accessed?

Reply:
thanks for spotting that! Obviously a mistake. :)